pkg/router/controller/router_controller.go

```diff
@@ -1,12 +1,13 @@
 package controller
 
 import (
+	"context"
 	"fmt"
 	"sync"
 	"time"
 
 	kapi "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
This change looks unnecessary (the package name is already "v1").

Reply:
gofmt adds it to disambiguate from `routev1 "github.com/openshift/api/route/v1"`.

Reply:
Actually, it could be gopls import organization. In any case, why not be explicit?

Reply:
(I will change it to better represent the package)

Reply:
Changed the import to `metav1`